tools/mksurfdata_esmf/src/mkfileMod.F90

```diff
@@ -105,6 +105,7 @@ subroutine mkfile_define_atts(pioid, dynlanduse)
     character(len= 10)   :: time               ! temporary
     character(len=  5)   :: zone               ! temporary
     integer              :: rcode
+    real(r4), parameter  :: data_version = 5.3   !  Compatibility version number of the datasets to create
```

Comment:
Add comments that the version is only updated when changes to surface datasets make them incompatible with previous versions.